.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "editor.fontSize": 10
```

Comment:
Sollen solche Files mit im PR landen, oder soll der `.vscode`-Ordner in die `.gitignore` aufgenommen werden?

Reply:
nope . sry … muss in die .gitignore

Reply:
Alles klar. Würde den PR dann mergen wenn der Change drin ist.